packages/docs/src/lang/en/components/DatePickers.json

```
@@ -44,6 +44,10 @@
       "desc": "Date picker can now select multiple dates with the `multiple` prop. If using `multiple` then date picker expects its model to be an array.",
       "header": "### Date pickers - Mutiple"
     },
+    "date-range": {
+      "desc": "Date picker can select date range with the `range` property. When using `range` property date picker expects its model to be an array of length 2 or empty.",
```

Use `prop` for consistency here please.

```suggestion
      "desc": "Date picker can select date range with the `range` prop. When using `range` prop date picker expects its model to be an array of length 2 or empty.",
```